src/Cli/InstallCommand.php

```diff
@@ -21,6 +23,11 @@ protected function configure()
         ;
     }
 
+    public function setDockerInstaller(DockerInstaller $dockerInstaller)
```

Comment:
are you happy for the InstallCommand to have a different constructor signature to the parent? it's one of those things I'm never sure about.